Installing Busy on Ubuntu os

<!--
Thanks for opening an issue! To help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. 
-->

#### Expected behavior

<!-- What do you think should happen? -->

I am installing busy for the first time in ubuntu os - AWS. I tried following the steps:

git clone https://github.com/busyorg/busy.git
cd busy

yarn
yarn start

#### Actual behavior

<!-- What actually happens? -->
When i give "yarn start" i get this error below:

ubuntu@ip-172-31-26-9:~/busy$ yarn start
yarn run v1.7.0
$ node ./build/server.js
module.js:549
    throw err;
    ^

Error: Cannot find module '/home/ubuntu/busy/build/server.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

#### How to reproduce

git clone https://github.com/busyorg/busy.git
cd busy

yarn
yarn start

<!-- For bugs, provide an URL that demos the problem -->

* Browser: 
* Operating system: Ubuntu 16.04.4 LTS (GNU/Linux 4.4.0-1060-aws x86_64)
<img width="572" alt="screenshot at may 26 19-40-12" src="https://user-images.githubusercontent.com/13276642/40577035-a3e5da20-611c-11e8-8d2f-3d7e2fc1ef47.png">

--- a/README.md
+++ b/README.md
@@ -16,11 +16,22 @@ For more info: https://busy.org/@busy.org
 
 ## Quick Setup
 
-```
+```bash
 git clone https://github.com/busyorg/busy.git
 cd busy
 
 yarn
+yarn dev
+```
+
+## Production build
+
+```bash
+git clone https://github.com/busyorg/busy.git
+cd busy
+
+yarn
+yarn build
 yarn start
 ```
 

--- a/docs/CONTRIBUTING.md
+++ b/docs/CONTRIBUTING.md
@@ -72,9 +72,10 @@ let us know in the issue so we can mark it as In progress.
    `git clone https://github.com/busyorg/busy.git`
 
 3. Install dependencies: `yarn`
-4. Create branch for your changes: `git checkout -b something-amazing`
-5. Commit your changes.
-6. Create a [pull request](https://github.com/busyorg/busy/compare).
+4. Start the server: `yarn dev`
+5. Create branch for your changes: `git checkout -b something-amazing`
+6. Commit your changes.
+7. Create a [pull request](https://github.com/busyorg/busy/compare).
 
 If you have problems setting up you might find those resources helpful:
 

